packages/fuels-core/src/code_gen/abigen.rs

```diff
@@ -260,6 +312,32 @@ impl Abigen {
     pub fn get_types(abi: &ProgramABI) -> HashMap<usize, TypeDeclaration> {
         abi.types.iter().map(|t| (t.type_id, t.clone())).collect()
     }
+
+    pub fn resolve_logs(&self) -> Result<Vec<TokenStream>, Error> {
```

Comment:
This doesn't seem to return Result::Err anywhere. 

Maybe return just a `Vec<TokenStream>` ?